Client/Helpers/TabEventHandler.swift

```diff
@@ -76,6 +77,7 @@ extension TabEventHandler {
     func tabDidGainFocus(_ tab: Tab) {}
     func tabDidLoseFocus(_ tab: Tab) {}
     func tabDidClose(_ tab: Tab) {}
+    func tabDidChangeContentBlockerStatus(_ tab: Tab) {}
```

Comment:
`didFinishNavigation`, but as mentioned, am very concerned it will work

Reply:
I've changed how this event is fired. The name `tabDidChangeContentBlockerStatus` fits better now.